ossrelease/msgs/feature_branch_complete

```diff
@@ -0,0 +1,14 @@
+The branch creation and feature freeze for our upcoming _salt_version_ release is now complete. If your PR was merged into develop before today or has an _salt_version_ label applied to the PR it will be included in the _salt_version_ release. We will review the PRs submitted before today again this week to ensure we did not miss any needed features for _salt_version_. We will also be making efforts this week to merge the remaining PRs with _salt_version_ labels. We are no longer accepting new features for the _salt_version_ release.
+
+If you have a question about a particular PR submitted please email the Release Team at open-release-team@saltstack.com
+
+Critical bug fixes for _salt_version_ should now be committed to the "_salt_version_" branch. Bug fixes that were previously submitted on the "develop" branch should be moved to the "_salt_version_" branch.
```

Comment:
I think the last sentence in this paragraph is confusing and probably not needed.

I know we make an effort to move bug-fixes from `develop` to the new branch right after branching, and bug fixes should be made the same way they always are: against the branch that needs the fix. Therefore, I think that sentence should be removed. Thoughts?